### PR TITLE
ML-144 first experiment, gru

### PR DESCRIPTION
**Pull request by enevoldsen1oo**

### CODE REVIEW

File: core/notebooks/ml_example_pipeline.ipynb
Comment (Tinggaard) [inline core/notebooks/ml_example_pipeline.ipynb:1]: Please revert this file... :)

File: core/notebooks/01_trefor-park-gru.ipynb
Comment (Tinggaard) [inline core/notebooks/01_trefor-park-gru.ipynb:1]: Would be nice if cells containing only comments were changed to markdown :)

**Review by Tinggaard — CHANGES_REQUESTED** [review]

File: core/notebooks/01_trefor-park-gru.ipynb
Comment (Thomast2001) [inline core/notebooks/01_trefor-park-gru.ipynb:1]: fix """todo""" nu når du er i gang :)

**Review by Tinggaard — APPROVED** [review]
Very nice job, good thing you resolved those conflicts, thank you so much, you are the best!

**Pull request merged**